packages/components/src/docs/General/Accessibility.mdx

```diff
@@ -0,0 +1,74 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Docs/General/Accessibility" />
+
+# Accessibility
+
+Union Investment strives to offer digital products and services that are accessible to all, regardless of their abilities/limitations or the technology used. This is an accessibility checklist used by the SDS team when designing and developing new features. It helps us improve the experience for everyone who uses our product and ensures our high standards are met.
+
+This is an ongoing process. We will continue to improve accessibility and meet at least the **WCAG 2.1 Level AA guidelines** - also with a view to the Barrier-Free Accessibility Strengthening Act that will come into force from 2025.
+
+## Developing a WCAG-compliant Basis
+
+We are committed to creating products that conform to the WCAG 2.1 guidelines, a standard also embraced by AK CD. This commitment includes a range of specific practices:
+
+### Semantic Structure & HTML Usage
+
+- **Native HTML over ARIA labels**: Always use native HTML instead of ARIA labels. This practice ensures that our content is as accessible as possible to a wide range of assistive technologies.
+
+- **Semantically Correct Structure**: Implement a semantically correct structure, for example, H1 followed by H2, and P used for text, etc. This helps assistive technologies understand the structure and meaning of our content.
+
+### Screen reader & keyboard controls
+
+- **Keyboard-Accessible Elements**: All elements should be focusable only with the keyboard, in a desired order and with a visual focus. This assures that our content is accessible to users who rely on keyboard navigation.
+
+- **Preserving ARIA Information**: Do not delete the ARIA information from the solid examples. They ensure correct interpretation by screen readers.
+
+### Multimedia:
+
+- **Alt text**: Provide alternative text for images, icons, and non-text content.
+
+- **Captions and transcripts**: Always include captions and transcripts for videos and audio content.
+
+- **Avoid auto-playing** audio and video without user controls.
```

Comment:
I am not sure about this one. AFAIK (but pretty old information), autoplay is not allowed. Firefox even banned it for a while from the browser.

If so, we should rephrase this to something like "Do not use auto-playing"